bigbluebutton-html5/package.json

```diff
@@ -19,6 +19,7 @@
     "meteor-node-stubs": "^0.2.3",
     "node-sass": "~3.8.0",
     "probe-image-size": "~2.1.1",
+    "react-addons-shallow-compare": "^15.4.2",
```

Comment:
I can't find where you used this package. Probably forgot to delete..
It is a legacy add-on, by the way :-)